apps/clearance_ui/src/components/clearance_library/bulk_conclusions/DataTable.tsx

```diff
@@ -56,6 +58,42 @@ export function DataTable<TData, TValue>({
         pageCount: pageCount,
         getCoreRowModel: getCoreRowModel(),
         manualPagination: true,
+        meta: {
+            editedRows,
+            setEditedRows,
+            revertData: (rowId: number, revert: boolean) => {
+                if (revert) {
+                    setData((old) =>
+                        old.map((row, index) =>
+                            index === rowId ? initialData[rowId] : row,
+                        ),
+                    );
+                } else {
+                    setData((old) =>
+                        old.map((row, index) =>
+                            index === rowId ? data[rowId] : row,
+                        ),
+                    );
+                }
+            },
```

Comment:
What does this do? A comment would be great, especially as it's a `revertData` function that can take `revert=false` as an argument, which is not immediately obvious.

Reply:
Separated this to revertData and updateOriginalData functions for better readability